web/src/helpers/featureFlags.js

```diff
@@ -0,0 +1,7 @@
+export const isRemoteParam = () => {
+  return new URLSearchParams(window.location.search).get('remote') === 'true';
+};
+
+export const isAggregatedViewEnabled = () => {
+  return new URLSearchParams(window.location.search).get('aggregatedView') === 'true';
+};
```

Comment:
I think we should use the useFeatureToggle hook that is already in place

If we want to ensure that we have everything in one place we can still keep the strings located in this file, eg

useFeatureToggle(AGGREGATED_VIEW)

Note: the remote flag is not considered a feature-flag at least in the way we've worked with the flag in the past.

Reply:
TODO remove the feature flag hook, hooks can only be used within components so when setting redux data the hook doesn't have context